test/integration/policy_multiline_templatization_test.go

```diff
@@ -68,43 +67,31 @@ var _ = Describe(
 			common.DoRootComplianceTest(policyHubName, policiesv1.Compliant)
 		})
 
-		It("The ConfigMaps on the hub cluster should be patched with the correct data", func() {
-			By("Checking the edited configMaps")
-			Eventually(
-				func() string {
-					configMap, err := clientHub.CoreV1().ConfigMaps(userNamespace).Get(
-						ctx, configMapName1, metav1.GetOptions{},
-					)
-					if err != nil {
-						return ""
-					}
-
-					return configMap.Data["extraData"]
-				},
-				defaultTimeoutSeconds*2,
-				1,
-			).Should(Equal("exists!"))
-			Eventually(
-				func() string {
-					configMap, err := clientHub.CoreV1().ConfigMaps(userNamespace).Get(
-						ctx, configMapName2, metav1.GetOptions{},
-					)
-					if err != nil {
-						return ""
-					}
-
-					return configMap.Data["extraData"]
-				},
-				defaultTimeoutSeconds*2,
-				1,
-			).Should(Equal("exists!"))
+		It("The ConfigMaps on the managed cluster should be patched with the correct data", func() {
```

Comment:
Should this also verify the Hub results as well?

Reply:
I don't think so because there's no guarantee that the hub is self-managed.

Reply:
It's an interesting point. I'll add in the check, but it does make me wonder whether we ever test a non-self-managed hub with a managed cluster. I guess not...